packages/core/src/domain/console/consoleObservable.ts

```diff
@@ -57,21 +57,23 @@ function createConsoleObservable(api: ConsoleApiName) {
 }
 
 function buildConsoleLog(params: unknown[], api: ConsoleApiName, handlingStack: string): ConsoleLog {
-  const log: ConsoleLog = {
-    message: [`console ${api}:` as unknown]
-      .concat(params)
-      .map((param) => formatConsoleParameters(param))
-      .join(' '),
-    api,
-  }
+  // Todo: remove console error prefix in the next major version
+  const prefix = api === ConsoleApiName.error ? `console ${api}: ` : ''
+  let message = params.map((param) => formatConsoleParameters(param)).join(' ')
+  let stack
 
   if (api === ConsoleApiName.error) {
     const firstErrorParam = find(params, (param: unknown): param is Error => param instanceof Error)
-    log.stack = firstErrorParam ? toStackTraceString(computeStackTrace(firstErrorParam)) : undefined
-    log.handlingStack = handlingStack
+    stack = firstErrorParam ? toStackTraceString(computeStackTrace(firstErrorParam)) : undefined
+    message = `${prefix}${message}`
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏simply add the prefix here, since it's the same condition
```suggestion
    message = `console error: ${message}`
```

Reply:
My bad, indeed :)